### PR TITLE
Added domain message and circle ci

### PR DESCRIPTION
**Pull request by alexander-schranz**
There should be a general event class be available.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "require": {
         "php": "^7.1",
         "ext-redis": "^4.2",
-        "symfony/messenger": "^4.1",
+        "symfony/messenger": "^4.2@RC || ^4.2",
```
Comment (wachterjohannes) [inline composer.json]: `^4.2@RC` includes `^4.2`

File: Message/DomainEventMessage.php [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+    /**
+     * @var string
+     */
+    protected $created;
+
+    public function __construct(string $name, string $type, string $id, array $payload = [])
+    {
+        $this->name = $name;
+        $this->type = $type;
```
Comment (wachterjohannes) [inline Message/DomainEventMessage.php:37]: should we add some default types `created`, `modified`, ... to constants?

**Review by wachterjohannes — CHANGES_REQUESTED** [review]

File: Message/DomainEventMessage.php (hunk 2 above)
Comment (alexander-schranz) [reply Message/DomainEventMessage.php:37]: no events are implementation specific `contact.created`

File: composer.json (hunk 1 above)
Comment (alexander-schranz) [reply composer.json]: updated

**Pull request merged**